_posts/op-ed/2022-11-01-What-is-High-Priority.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: What is High Priority?
+description: >
+  "That doesn't matter!" – it's said far too often, isn't it? But did you know that we often don't consider facts and conditions when we think about priorities?
+date: 2022-11-01 16:00:00 +0100
+image: '/images/posts/prism-crystal.jpg'
+tags: [op-ed, time, management, organization]
+featured: false
+---
+
+### The <font color="#F88">Time Management</font> Series
+
+This post is just one part of my [Time Management](/2022/11/01/Time-is-of-the-Essence) series. I really recommend you read the whole series if you have time, but I'll do my best to make this part self-contained.  
+**Important disclaimers** are also included there.
+
+-----
+
+## A bullet-proof framework
+
+#### 🎯 &nbsp; About goals…
+
+More controversial opinions: ✅
+
+There are a few things I consider when setting goals for myself and my organization[s]:
+
+1. The _more responsibility_ I bear for my decisions, the _less_ often I have to make decisions. Make sure those choices are good 
+2. I can do things right in many different ways, often _without_ going through the _known/proven steps_. Skip steps when possible 
+3. I've become _brutally honest_ with myself about my capacity and time commitment. This is to the point that I _give up_ on most (90%) of my ideas and external requests. This is good, be transparent and reject things
+
+**What do you disagree with the most?!**
+
+Either way, I think I can justify most of the things I just wrote, even though my framework isn't great. 😄 But it's a good starting point if we're looking for a reliable algorithm to prioritize tasks, I think.
+
+So what are we looking for?
+
+```kotlin
+fun findPriorityItem(items: List<WorkItem>): WorkItem {
+  // FIXME: This is what we're looking for
+  error("Too hard to implement")
+}
+```
+
+#### 🌱 &nbsp; The basic framework
+
+Visualizations are nice, but let's also clarify. Remember that all classifications here are **relative** and not absolute, we're always comparing work items to other work items on the table.
+
+{:refdef: style="text-align: center;"}
+![Priority Matrix](/images/posts/time-management-priority-table.png)
+*The Priority Matrix (Nike don't sue me)*
```

Comment:
This is a variation of the Eisenhower (decision) matrix/Priority matrix which you should reference. It's a common tool for prioritization and many people will be familiar with it, so you can use it as a starting point and explain how your ideas differ